src/shell/Aliases.ts

```diff
@@ -4,9 +4,12 @@ import * as _ from "lodash";
 export const aliasesFromConfig: Dictionary<string> = {};
 
 export async function loadAliasesFromConfig(): Promise<void> {
-    const lines = await executeCommandWithShellConfig("alias");
-
-    lines.map(parseAlias).forEach(parsed => aliasesFromConfig[parsed.name] = parsed.value);
+    try {
+        //const lines = await executeCommandWithShellConfig("alias");
+        //lines.map(parseAlias).forEach(parsed => aliasesFromConfig[parsed.name] = parsed.value);
+    } catch (error) {
+        console.error(error);
+    }
```

Comment:
Windows PowerShell has aliasing feature which you can know more
by opening a PowerShell, then run
```powershell
Get-Help about_Aliases
```